kratos/processes/process.h

```diff
@@ -168,6 +168,16 @@ class Process : public Flags
         return 0;
     }
 
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     */
+    virtual const Parameters GetDefaultParameters() const
+    {
+        KRATOS_WARNING("Process") << "GetDefaultParameters not implemented. Returning empty parameters" << std::endl;
```

Comment:
I'm wondering if we should throw a meaningful error in here.

Consider that if we get here it is possibly because someone is trying to get the defaults to do the check but no custom defaults are implemented in the derived process. Hence, an error will be thrown in any case,

Reply:
As @KratosMultiphysics/technical-committee  decides

Reply:
@KratosMultiphysics/technical-committee opts for throwing an error in this case fostering the implementation in the derived class in the future